source/user-manual/capabilities/file-integrity/how-it-works.rst

```diff
@@ -55,3 +45,26 @@ How it works
 			- MD5: 146c07ef2479cedcd54c7c2af5cf3a80
 			- SHA1: e1efc99729beb17560e02d1f5c15a42a985fe42c
 			- SHA256: a7998f247bd965694ff227fa325c81169a07471a8b6808d3e002a486c4e65975
+
+By default, each Wazuh agent has the syscheck enabled and preconfigured but it is recommended to review and amend the configuration of the monitored host.
+
+File integrity monitoring results for the whole environment can be observed in :ref:`Kibana app<kibana_app>` in the *Overview > Integrity monitoring* section:
+
+.. thumbnail:: ../../../images/manual/fim/fim-kibana-overview1.png
+  :title: Kibana app: Fim overview
+  :align: center
+  :width: 100%
+
+.. thumbnail:: ../../../images/manual/fim/fim-kibana-overview2.png
+  :title: Kibana app: Fim overview2
+  :align: center
+  :width: 100%
+
+.. thumbnail:: ../../../images/manual/fim/fim-kibana-overview3.png
+  :title: Kibana app: Fim overview3
+  :align: center
+  :width: 100%
+
+Due to the changes made on the newest FIM, only the Wazuh agents which are in version starting from 3.12 will use new functionality. All the Wazuh agents below that version will keep reporting syscheck scan results as in the older versions. If the Wazuh agent is upgraded to the v3.12 and keeps its old configuration, will still use the new FIM module and the default synchronization settings.
+
+After the Wazuh agent service restart, the FIM database is cleared. Changes in the monitored directories done while the service is not running will not be reported to the Wazuh manager. The same will happen to the changes that occur after the last scan and before the restart, the synchronization will not take place on the directories that are not monitored with the ``realtime`` or with the ``whodata``.
```

Comment:
At the end of this page, we need to drive the users to the configuration page. Something like `Please, see the following document to see all available configuration options` or similar.

Reply:
Done